plugins/modules/rpm_ostree_pkg.py

```diff
@@ -126,8 +161,21 @@ def ensure(self):
             results['action'] = 'uninstall'
             cmd.append('uninstall')
 
+        if self.allow_inactive:
+            cmd.append('--allow-inactive')
+        
+        if self.idempotent:
+            cmd.append('--idempotent')
```

Comment:
Why would we want to make idempotency optional? The usual recommendation is to always have the modules being idempotent.

Reply:
That's true, i will change this